src/main/java/chess/dao/SquareDao.java

```diff
@@ -0,0 +1,78 @@
+package chess.dao;
+
+import chess.DBConnection;
+import chess.dto.BoardDto;
+import chess.dto.PieceDto;
+import chess.dto.PositionDto;
+import chess.dto.SquareDto;
+import chess.dto.SquaresDto;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+
+public class SquareDao {
+
+    public void insertSquare(final int boardId, final PositionDto positionDto, final PieceDto pieceDto) {
+        final String query = "INSERT INTO squares VALUES(?, ?, ?, ?, ?)";
+
+        try (final Connection connection = DBConnection.getConnection();
+             final PreparedStatement preparedStatement = connection.prepareStatement(query, Statement.RETURN_GENERATED_KEYS)) {
+            preparedStatement.setInt(1, boardId);
+            preparedStatement.setInt(2, positionDto.fileIndex());
+            preparedStatement.setInt(3, positionDto.rankIndex());
+            preparedStatement.setString(4, pieceDto.color());
+            preparedStatement.setString(5, pieceDto.pieceType());
+
+            preparedStatement.executeUpdate();
+        } catch (final SQLException e) {
+            throw new IllegalStateException("[ERROR] 기물과 위치를 저장하는 도중 오류가 발생하였습니다 : " + e.getMessage());
+        }
+    }
+
+    public void deleteAllSquares(final BoardDto boardDto) {
```

Comment:
```suggestion
    public void deleteAllById(final int boardId) {
```
여기도 findById처럼 boardId만 받으면 되지 않을까요?